pyvista/core/dataset.py

```diff
@@ -553,7 +553,7 @@ def _activate_texture(mesh, name: str) -> _vtk.vtkTexture:
         try:
             texture = mesh.textures[name]
         except KeyError:
-            warnings.warn(f'Texture ({name}) not associated with this dataset')
+            warnings.warn(f'Texture ({name!r}) not associated with this dataset')
```

Comment:
This is just a matter of style, but e.g. in the "invalid" test seeing "`Texture ('not a key') not associated ...`" is better than "``Texture (not a key) not associated ..." in my opinion

Reply:
This seems slightly better to me.